cmd/frontend/graphqlbackend/saved_searches.go

```diff
@@ -111,20 +112,106 @@ func (r *schemaResolver) toSavedSearchResolver(entry types.SavedSearch) *savedSe
 	return &savedSearchResolver{db: r.db, s: entry}
 }
 
-func (r *schemaResolver) SavedSearches(ctx context.Context) ([]*savedSearchResolver, error) {
+func (r *schemaResolver) SavedSearches(ctx context.Context, args struct {
+	Namespace graphql.ID
+	First     *int32
+	Last      *int32
+	After     *string
+	Before    *string
+}) (*graphqlutil.ConnectionResolver[savedSearchResolver], error) {
```

Comment:
You can embed this struct:

```suggestion
func (r *schemaResolver) SavedSearches(ctx context.Context, args struct {
	graphqlutil.ConnectionResolverArgs
	Namespace graphql.ID
}) (*graphqlutil.ConnectionResolver[savedSearchResolver], error) {
```

Reply:
Nice to be reminded about this feature